.github/workflows/publish-oss-for-cloud.yml

```diff
@@ -144,3 +157,29 @@ jobs:
       - name: Cleanup Docker buildx
         run: docker buildx rm oss-buildx
         shell: bash
+
+  stop-runner:
```

Comment:
oh did we previously not stop runners?

Reply:
yup, seems like a bug that went unnoticed